trieste/acquisition/function/function.py

```diff
@@ -232,8 +227,28 @@ def __call__(self, x: TensorType) -> TensorType:
         return (self._eta - mean) * normal.cdf(self._eta) + variance * normal.prob(self._eta)
 
 
-class log_expected_improvement(AcquisitionFunctionClass):
-    def __init__(self, model: ProbabilisticModel, eta: TensorType):
+class LogExpectedImprovement(ExpectedImprovement):
+    """
+    Builder for the log expected improvement function.
```

Comment:
needs some more explanation what is this, reference, perhaps also why to use instead of the original, as a guidence to users